fidouafclient/app/src/main/java/org/ebayopensource/fidouafclient/op/Reg.java

```diff
@@ -38,17 +38,22 @@
 public class Reg {
 	
 	private Gson gson = new GsonBuilder().disableHtmlEscaping().create();
-	
+
 	public String getUafMsgRegRequest (String username, String appId){
```

Comment:
I believe that argument "appId" is not passed as app Id. The actual value that is coming here is the facet Id. It is passed from here:
https://github.com/eBay/UAF/blob/a9a14fe51eae6b4d8c319b965fb953b539aadcd1/fidouafclient/app/src/main/java/org/ebayopensource/fidouafclient/MainActivity.java#L132

The "MainActivity" would also need to change.

Did I miss something?